app/forms/placements/placements/filter_form.rb

```diff
@@ -4,7 +4,7 @@ class Placements::Placements::FilterForm < ApplicationForm
   attribute :school_ids, default: []
   attribute :subject_ids, default: []
   attribute :school_types, default: []
-  attribute :partner_school_ids, default: []
+  attribute :partner_schools, default: []
```

Comment:
This could become a boolean and be given a more descriptive name.

```suggestion
  attribute :only_partner_schools, :boolean, default: false
```